qa/rpc-tests/README.md

```diff
@@ -35,8 +30,8 @@ If you set the environment variable `PYTHON_DEBUG=1` you will get some debug out
 
 A 200-block -regtest blockchain and wallets for four nodes
 is created the first time a regression test is run and
-is stored in the cache/ directory. Each node has 25 mature
-blocks (25*50=1250 BTC) in its wallet.
+is stored in the cache/ directory.  Each node has the miner
```

Comment:
Nit: Don't change the whitespace convention used by a file without good reason. This will not match future changes to this file in any upstream PRs we pull in.